joint_state_controller:
  type: "joint_state_controller/JointStateController"
  publish_rate: 50

left_wheel_joint_velocity_controller:
  type: "velocity_controllers/JointVelocityController"
  joint: "banhtrai_joint"
  pid: {p: 100.0, i: 0.1, d: 10.0}

right_wheel_joint_velocity_controller:
  type: "velocity_controllers/JointVelocityController"
  joint: "banhphai_joint"
  pid: {p: 100.0, i: 0.1, d: 10.0}

front_wheel_joint_velocity_controller:
  type: "velocity_controllers/JointVelocityController"
  joint: "banhtruoc_joint"
  pid: {p: 100.0, i: 0.1, d: 10.0}

arm_1_joint_controller:
  type: "position_controllers/JointPositionController"
  joint: "prismatic_joint"
  pid: {p: 100.0, i: 0.05, d: 1.0}

arm_2_joint_controller:
  type: "position_controllers/JointPositionController"
  joint: "revolute_joint"
  pid: {p: 100.0, i: 0.05, d: 1.0}
